### PR TITLE
Make Resource conversion tests also check params and tags

### PR DESCRIPTION
**Pull request by lucastt**
This changes should fix the current behavior of golden file tests that do not really check important parameters and only check the correctness of the template, which is not really testing the stack generation algorithm of the ingress controller.

https://github.com/zalando-incubator/kube-ingress-aws-controller/issues/626

### CODE REVIEW

File: aws/fake/cf.go [hunk 1]
```diff
@@ -52,7 +62,10 @@ func (m *CFClient) DescribeStacks(in *cloudformation.DescribeStacksInput) (*clou
 }
 
 func (m *CFClient) CreateStack(params *cloudformation.CreateStackInput) (*cloudformation.CreateStackOutput, error) {
-	m.lastGeneratedTemplate = *params.TemplateBody
+	m.lastStackTags = params.Tags
+	m.lastStackParams = params.Parameters
+	m.lastStackTemplate = *params.TemplateBody
```
Comment (AlexanderYastrebov) [inline aws/fake/cf.go:67]: Could params.TemplateBody be nil? Maybe its safer to use pointer as well.

Comment (AlexanderYastrebov) [reply aws/fake/cf.go:67]: Ah, I see it was dereferenced in the existing code.

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
Marked as do-not-merge until we decide if we merge to master or a feature branch

File: aws/fake/cf.go (hunk 1 above)
Comment (lucastt) [reply aws/fake/cf.go:67]: It should not be possible, if this is the case this should be an error.

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
@AlexanderYastrebov I think in this case there is only test changed, so no production code path can change so we can also merge it to master IMO.

**Pull request merged**